python/pyspark/tests.py

```diff
@@ -543,6 +543,15 @@ def test_tc_on_driver(self):
         tc = TaskContext.get()
         self.assertTrue(tc is None)
 
+    def test_get_local_property(self):
+        """Verify that local properties set on the driver are available in TaskContext."""
+        self.sc.setLocalProperty("testkey", "testvalue")
```

Comment:
We should cleanup the property after the test to avoid affecting to other tests?

Reply:
there isnt a clear way to clear local property in the spark context? what do you propose is the right approach here? set the local property as "None"?